packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -595,7 +594,8 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
   }
 
   /** Emit an event that triggers a UI re-render */
-  private emitTorrentUpdated(infoHash) {
+  private emitTorrentUpdated(infoHash, trigger) {
+    // log.trace(`emitTorrentUpdate: ${trigger}`);
```

Comment:
Does this comment belong?

Reply:
I would be in favor of leaving this line in here. The intention is to make it easier to "profile" why rerendering occurs.